code/index.html

```diff
@@ -3,28 +3,162 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link  href="./style.css" rel="stylesheet"  type="text/css">
+    <link rel="stylesheet" href="https://use.typekit.net/lml3gyj.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="apple-touch-icon" sizes="180x180" href="./favicon_io/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./favicon_io/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./favicon_io/favicon-16x16.png">
+    <link rel="manifest" href="./favicon_io/site.webmanifest">
+  <title>EDGE</title>
   </head>
+  
   <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
+    <header id="hero"> <!-- Hero banner with video -->
+      <!--Video by Arvind Balagopal on Pexels-->
+      <video loop muted autoplay preload="auto">
+        <source src="./videos/hero-video_compressed.mp4" type="video/mp4">
+        <p>Din webbläsare har inte stöd för video.</p>
+      </video>
+      
+      <div class="caption">
+        <h1>EDGE</h1>
+        <p>FEMALE NETWORK</p>
+        <p>EMPOWER • DRIVEN • GOALS • ENERGY</p>
+      </div>
+    </header>
+    
+    <div class="page-content"> <!-- Page content -->
+      <h2> Varmt välkommen till EDGE! </h2>
+      <p> Powernätverket för kvinnor i Jönköping. </p>
+      <p> <i>* Some image gallery would be fun here :) * </i></p>
+      <br>
+      <img src="./img/Logga_EDGE_svart.png" class="photo"/>
+      <img src="./img/web_A6A4663.jpg" class="photo"/>
```

Comment:
Nice to see file structure with an image folder, but it's a must to use the 'alt' attribute for images. Here's some reading: 
https://www.w3schools.com/tags/att_img_alt.asp